test/index.test.js

```diff
@@ -36,6 +36,15 @@ test('@transform-disable', () => {
 	expect(result2.code).toBe(code2)
 })
 
+test('template require', () => {
+	var code = "require('./test-template.html')";
+	var result = babel.transform("Vue.component({template:" + code + "});", options())
+	expect(result.code).toBe("Vue.component({render(){return function(){var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _vm._m(0);}.call(this);},staticRenderFns:[function(){var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',[_c('section',[_vm._v(\"Test Template\")])]);}]});")
```

Comment:
I guess we can start using `expect().toMatchSnapshot()` to prevent from manually copy/paste these code.